src/styles/base/global.scss

```diff
@@ -61,3 +62,57 @@ figure {
     padding-inline: 0;
   }
 }
+
+@include laptop {
+  .layout-container {
+    max-width: 1728px;
+    margin: 0 auto;
+    padding-inline: 64px;
+
+    &--padding-large {
+      padding-inline: 118px;
+    }
+
+    &--padding-none {
+      padding-inline: 0;
+    }
+  }
+}
+
+@include tablet {
+  .layout-container {
```

Comment:
I hope you're understand what happens on this screen, but if not, lets clarify it! It's important part to debbug our code throw dev tools. Here is example how we can find dublicate our styles

<img width="582" alt="image" src="https://github.com/user-attachments/assets/3bcce2fe-60ae-499f-9de5-bf7991e9b5ec" />